guides/07_FAQs.md

```diff
@@ -122,6 +122,10 @@ The Amazon Chime SDK does not support scheduling meetings ahead of time. The mom
 [CreateMeetingWithAttendees](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateMeetingWithAttendees.html) is invoked and the meeting is created, the auto end
  policies will kick in and if no one joins the meeting, it will close within 5 minutes. The application can create the meeting when drawing closer to the meeting start time or when the the first attendee is ready to join.
 
+ ### What happens when I try to re-use same attendee response to join a meeting twice?
+ 
+ Suppose an attendee has joined a meeting, now if that same attendee tries to join the same meeting again using the same `AttendeeId` or `ExternalUserId` response received from [CreateAttendee](https://docs.aws.amazon.com/chime/latest/APIReference/API_CreateAttendee.html) API, then, the first attendee will automatically leave the meeting with [`AudioJoinFromAnotherDevice`](https://aws.github.io/amazon-chime-sdk-js/enums/meetingsessionstatuscode.html#audiojoinedfromanotherdevice) meeting session status code. Reference issue: [#1290](https://github.com/aws/amazon-chime-sdk-js/issues/1290). The `AudioJoinFromAnotherDevice` meeting session status code is triggered by Amazon Chime backend and it is not triggered internally from the JS SDK.
```

Comment:
Would it be better change `response received` to `and receives response`, like 
```
same attendee tries to join the same meeting again using the same `AttendeeId` or `ExternalUserId` and receives response from CreateAttendee API

Reply:
Actually, we join the meeting after we have received the response from the `CreateAttendee` API so kept it in past tense. I am updating the wordings a little. Thanks for your review!